### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Wilke000**
# Changes
**Adds simulate unlock clarity**

### CODE REVIEW

**Review by VillainsRule4000 — CHANGES_REQUESTED** [review]
could use clarity on "what you could open that day"

File: README.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ There are 3 good methods to using these scripts:
...
 </details>
 
+<details><summary><h3>Why can't I open some blooks with simulate unlock?</h3></summary>
+ 
+You can only simulate unlock what you can open that day. For example you can only get whatever color astronaut and the chromas in the permenant packs.
+</details>
+
 <details><summary><h3><script> is not working?</h3></summary>
 
 Make sure you're running it properly (see [How to use](#how-to-use)), if it still doesn't work and other cheats do, then [make an issue](https://github.com/Minesraft2/Blooket-Cheats/issues)
```
Comment (LeozaCEO2) [inline README.md:33]: I made an enhancement issue to write only blooks available at the current time can be simulated

Comment (VillainsRule4000) [reply README.md:33]: this pr is because of that issue.

**Comment by Wilke000** [comment]
> could use clarity on "what you could open that day"

Done

**Review by VillainsRule4000 — CHANGES_REQUESTED** [review]
this still doesn't clarify "what you can open that day" - change that language
also mysticals, not mythicals

**Comment by Wilke000** [comment]
Got it

**Review by VillainsRule4000 — APPROVED** [review]
I guess this works, Minesraft will likely commit this himself.

**Comment by Wilke000** [comment]
K :)

**Comment by LeozaCEO2** [comment]
If this doesn't work, which hopefully it will, just make a note that it only allows what is available in the Blooket shop at the the current time

File: README.md (hunk 1 above)
Comment (LeozaCEO2) [reply README.md:33]: yes ik

**Review by fowntain — APPROVED** [review]
I approve of these changes. +1

**Comment by Wilke000** [comment]
Fixed typo

**Review by LeozaCEO2 — APPROVED** [review]

**Comment by Greenlio** [comment]
if he wants to add it, he'll commit it himself. either way, this isn't getting merged.

**Comment by Wilke000** [comment]
> if he wants to add it, he'll commit it himself. either way, this isn't getting merged.

why not

**Comment by Greenlio** [comment]
> > if he wants to add it, he'll commit it himself. either way, this isn't getting merged.
> 
> why not

we don't merge 99% of pull requests because we don't want to add contributors

**Comment by Wilke000** [comment]
> > > if he wants to add it, he'll commit it himself. either way, this isn't getting merged.
> > 
> > 
> > why not
> 
> we don't merge 99% of pull requests because we don't want to add contributors

ah ok

**Pull request closed without merging**